Patches/DefaultCharacterDevelopmentModelPatch.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem.SandBox.GameComponents;
+
+namespace BannerlordTweaks.Patches
+{
+    [HarmonyPatch(typeof(DefaultCharacterDevelopmentModel), "get_LevelsPerAttributePoint")]
```

Comment:
Switch this to using an injected TweakedModel instead.
Have a look at the TweakedPregnancyModel.
Afterwards move out of the patches folder.

Reply:
You can create a folder Models and move all the Tweaked models there in a separate pull request.